src/commonMethods/commonFunctions.js

```diff
@@ -64,6 +90,74 @@ export const setWebKitUrl = function(URL) {
     .catch(err => err)
 }
 
+/**
+ * This function checks if the process is running by getting the process id and comparing it to the number.
+ *  - If the process id is a number, then false is returned
+ *  - If the process is not a number, then true is returned
+ * @param process
+ */
+export const checkIfProcessIsRunning = function(process) {
+  let opts = {
+    cmd: `ps w | grep ${process} | grep -v grep | awk ` + "'{printf(" + '"%i \\n\\r", $1)}' + "'",
+  }
+  exec(opts, res => {
```

Comment:
I guess this will not work in the browser version, as it's using a NodeJS only library (ssh2).
Care to chime in @wouterlucas ?